src/bundle/Resources/public/js/scripts/admin.format.date.js

```diff
@@ -1,4 +1,8 @@
-(function(moment) {
+(function(moment, eZ) {
+    const { backOfficeLanguage } = eZ.adminUiConfig;
+
+    moment.locale(backOfficeLanguage);
```

Comment:
Not sure, but maybe this should be set in the header in `src/bundle/Resources/views/themes/admin/ui/layout.html.twig` so that it will be set globally?

Reply:
Aren't we kinda setting it globally? This is the same file that adds additional methods to moment.js and is fired after loading dom?

Reply:
Ok, you are right, this file seems to fit this code. :)